### ISSUE TITLE
Coding standards

### ISSUE THREAD

**Issue by ivanoats** [orcasound/orcamap-react#10]
- eslint runs prettier
- css standard? 
- functional components and hooks over class based components
- types if possible

- anything else? comments and questions?

**Comment by GarrySan** [comment]
sass instead of css?

**Comment by ivanoats** [comment]
I don't think Sass is really necessary if we are using a css-in-js system styled components or css modules. It just adds an extra layer of tooling, and extra syntax to learn, in my opinion.

**Comment by GarrySan** [comment]
Yup, if we are using material-ui components then most of the styling will be of the css-in-js form, leaving very little components that we will need to style.

**Comment by ivanoats** [comment]
#19 takes care of most of this. The PR just needs conflicts resolved

**Comment by riteshsp2000** [comment]
Hey I had resolved the conflicts. But just saw now that other PRs were merged. I'll resolve the conflicts now and then it should be good to review

**Comment by ivanoats** [comment]
One we add these to https://github.com/orcasound/orcamap-react/blob/main/CONTRIBUTING.md we can close this issue.
